Controls/Compound.js

```diff
@@ -15,13 +15,8 @@
 //ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-define([
-        "require", "jquery", "_",
-        "AXM/AXMUtils", "AXM/DOM"],
-    function (
-        require, $, _,
-        AXMUtils, DOM) {
-
+define(["require", "jquery", "_", "AXM/AXMUtils", "AXM/DOM", "AXM/Units"],
+    function (require, $, _, AXMUtils, DOM) {
```

Comment:
Pardon my ignorance, but would this be the same?
```
function (require, $, _, AXMUtils, DOM, Unit) {
  // var Unit = require("AXM/Units");
...
```

Reply:
Yes, they are equivalent but the ergonomics aren't the same. I prefer to have the declaration on the same line as the require statement because of that mainly. 😄 

The name/path of the module can and often does differ from its definition. Having it on a single line makes it a breeze to correlate them. No need to go counting the items in the definition list, and then making my way back into the requires to find. It becomes especially cumbersome as the number of imports grows!

Also, it removes human error in messing up the order of the imports, because declaration and import share the same line. Deleting the import means deleting the entire line, that's it! There's no risk of human error where import and definition aren't matching up. I've seen it go unnoticed and break things in subtle ways :)